src/services/cache/CacheService.ts

```diff
@@ -79,6 +92,123 @@ export class CacheService {
     return this.dmrService.readResource(request);
   }
 
+  getCacheStats(container: ICacheContainer, cache: ICache): ng.IPromise<any> {
+    let firstServer: IServerAddress = container.serverGroup.members[0];
+    let address: string [] = [].concat("host", firstServer.host, "server", firstServer.name, "subsystem",
+      "datagrid-infinispan", "cache-container", container.name, cache.type, cache.name);
```

Comment:
But @ryanemerson but generateAddress is using profile path and here we need server path!